roles/ceph-common/defaults/main.yml

```diff
@@ -63,6 +63,13 @@ ceph_origin: 'upstream' # or 'distro'
 #
 ceph_use_distro_backports: false # DEBIAN ONLY
 
+ceph_mon_packages:
+  - ceph
```

Comment:
@leseb the ``tasks/vars/after_infernalis_vars.yml`` file is overriding those default values. For anything before jewel we still need ``ceph`` and ``ceph-mon`` or ``ceph-osd``.